docs/user-guide/introduction.md

```diff
@@ -29,7 +29,10 @@ It is probably not a good idea to deploy MicroMDM as a cost cutting option, as t
 # Apple Requirements
 
 If you've decided to run an instance of MicroMDM in your organization, there are a few Apple specific requirements you need to meet.  
-First, you need to [enroll](https://developer.apple.com/programs/enterprise/enroll/) your organization in the Apple Developer Enterprise Portal. Enrolling costs $299/year and requires that your organization have a [DUNS](https://en.wikipedia.org/wiki/Data_Universal_Numbering_System) number.  
+First, you need to [enroll](https://developer.apple.com/programs/enterprise/enroll/) your organization in the Apple Developer Enterprise Portal. Enrolling costs $299/year and requires that your organization have a [DUNS](https://en.wikipedia.org/wiki/Data_Universal_Numbering_System) number. 
+
+If you are an individual wanting to use/test MicroMDM you can [enroll](https://developer.apple.com/programs/enroll/) in Apple's standard Developer Account for $99/year. You won't be able to use Apple School/Business Manager or Automated Device Enrollment, but you can still have the `MDM CSR` option enabled in your account.
+
```

Comment:
I would collapse all of this and drop mention of the enterprise account, actually. Only really need to mention the standard account. The Enterprise account could be just a mention at the end or like you did below "Standard or Enterprise"

If we're mentioning the personal/test case, I might also through in https://mdmcert.download/ but could go either way there.